src/read.c

```diff
@@ -3390,6 +3394,29 @@ static avifDecoderItem * avifDecoderDataFindItem(avifDecoderData * data, avifBoo
     return NULL;
 }
 
+static avifResult avifDecoderGenerateImageTiles(avifDecoder * decoder, avifItemTiles * group, avifDecoderItem * item, avifBool alpha)
```

Comment:
Optional: Hmm... Given this function name, perhaps the first struct name I suggested, `avifImageTiles`, isn't bad after all.

The reason I thought `avifImageTiles` was a bad name is that an `avifImage` may consist of two HEIF image items (color and alpha). The name `avifImageTiles` could refer to an `avifImage` or a HEIF image item. So it could be ambiguous.